server/api/game/controller.js

```diff
@@ -61,11 +65,23 @@ class Game {
 
   endGame = () => {
     clearInterval(this.gameState.interval);
+    this.gameState.status = 2;
   };
 
-  moveGhosts = ({ ghosts, gridState, scatterStart }) => {
+  ifPacmanOnGhost = (ghost) => {
```

Comment:
This can be a pure function and moved to core.js

Reply:
This wasn't done in this PR. It was present here from the start.

Reply:
Will work on it in the next PR.

Reply:
Don't feel the need to make it pure.